modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template

```diff
@@ -1,7 +1,7 @@
-import { <%= classify(name) %> } from './<%= dasherize(name) %>.actions';
+import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
 
 describe('<%= classify(name) %>', () => {
   it('should create an instance', () => {
-    expect(new <%= classify(name) %>()).toBeTruthy();
+    expect(new from<%= classify(name)%>.Load<%= classify(name) %>s()).toBeTruthy();
```

Comment:
Can you add a test for this?

Reply:
ok sure. The new test will check for the content generated by the action spec schematics?

Reply:
Correct, for example https://github.com/ngrx/platform/blob/cdded0de149bea018b3760a791ed0a08b225ae34/modules/schematics/src/action/index.spec.ts#L102

Reply:
I have added the tests Tim. Can you check them ?

Reply:
```suggestion
    expect(new<%= classify(name)%>Actions.Load<%= classify(name) %>s()).toBeTruthy();
```